docs/tutorials/creating-your-first-substrate-chain/test.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Testing Your New Blockchain
```

Comment:
The title and filename are ambiguities, I was thought it related to testing code, not the UI.

Reply:
You do "test" the code with the front end right?

Could update though, I see what you mean

Reply:
Using Your New Blockchain
Interacting With Your New Blockchain